src/SDK/Client/Platform/methods/contracts/broadcast.ts

```diff
@@ -11,6 +11,8 @@ import { signStateTransition } from "../../signStateTransition";
  * @return dataContract
  */
 export default async function broadcast(this: Platform, dataContract: any, identity: any): Promise<any> {
+    await this.initialize();
```

Comment:
Let's remove `Client#initialize` method and call `this.dpp.initialize`?

Reply:
You mean platform#initialize?